core/src/main/scala/com/nvidia/spark/rapids/tool/tuning/AutoTuner.scala

```diff
@@ -1432,11 +1432,11 @@ class AutoTuner(
    * @return the property key to use for initial partition number
    */
   private def getInitialPartitionNumProperty: String = {
-    val minShufflePartitionsKey = "spark.sql.adaptive.shuffle.minNumPostShufflePartitions"
+    val maxNumPostShufflePartitions = "spark.sql.adaptive.maxNumPostShufflePartitions"
     val initialPartitionNumKey = "spark.sql.adaptive.coalescePartitions.initialPartitionNum"
-    // check if minShufflePartitionsKey is in final tuning table
-    if (finalTuningTable.contains(minShufflePartitionsKey)) {
-      minShufflePartitionsKey
+    // check if maxNumPostShufflePartitions is in final tuning table
+    if (finalTuningTable.contains(maxNumPostShufflePartitions)) {
+      maxNumPostShufflePartitions
```

Comment:
Seems we need to change Scala code for this alias of maxNumPostShufflePartitions and initialPartitionNum.
I suppose we should just adjust the target_cluster.yaml to define the alias without recompiling the tools core.
Do we need to add some Scala code if we have another alias case?

Reply:
Confirmed that this was a trade-off for a quick implementation. We should have a follow-up issue to complete the alias feature, which provides the capability to the user to customize the alias configs only by the yaml file.